google/cloud/devtools/containeranalysis_v1/types/containeranalysis.py

```diff
@@ -15,10 +15,75 @@
 # limitations under the License.
 #
 
+import proto  # type: ignore
+
+
+from grafeas.v1 import vulnerability_pb2 as vulnerability  # type: ignore
```

Comment:
ModuleNotFoundError: No module named 'grafeas.v1'. -- Should this be grafeas.grafeas_v1?